.vscode/extensions.json

```diff
@@ -2,6 +2,6 @@
 	// See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
It suggests anyone who pulls the repo to download the tslint extension so tslint errors will show in their editor.